com.gamestarmechanic.app.yaml

```diff
@@ -0,0 +1,77 @@
+---
+app-id: com.gamestarmechanic.app
+runtime: org.freedesktop.Platform
+runtime-version: '23.08'
+sdk: org.freedesktop.Sdk
+base: org.electronjs.Electron2.BaseApp
+base-version: '23.08'
+command: gamestarmechanic-wrapper
+separate-locales: false
+
+sdk-extensions:
+  - org.freedesktop.Sdk.Extension.node18
+  
+build-options:
+  append-path: /usr/lib/sdk/node18/bin
+  env:
+    NPM_CONFIG_LOGLEVEL: info
+
+finish-args:
+  - --device=dri
+  - --share=ipc
+  - --share=network
+  - --socket=pulseaudio
+  - --socket=x11
+  - --env=GAMESTAR_PLUGINS_PATH=/app/extra/plugins
+
+modules:
+  - name: gamestar-mechanic
+    buildsystem: simple
+    build-options:
+      env:
+        XDG_CACHE_HOME: /run/build/gamestar-mechanic/flatpak-node/cache
+    sources:
+      - type: git
+        url: https://github.com/dylanmccall/gamestar-mechanic-linux.git
+        branch: main
+      - generated-sources.json
+    build-commands:
+      # With yarn < 2, yarn config only writes to ~/.yarnrc and it can not be reasoned with.
+      - env HOME=$PWD yarn --offline config set yarn-offline-mirror "${FLATPAK_BUILDER_BUILDDIR}/flatpak-node/yarn-mirror"
+      - yarn --offline install --immutable --immutable-cache --inline-builds
+      - yarn --offline run pack --linux
+      # Bundle app and dependencies
+      - cp -a dist/linux*unpacked ${FLATPAK_DEST}/main
+      # Export app icon and desktop file
+      - install -m644 res/icon/com.gamestarmechanic.app.png -Dt ${FLATPAK_DEST}/share/icons/hicolor/512x512/apps
+      - install -m644 res/com.gamestarmechanic.app.desktop -Dt ${FLATPAK_DEST}/share/applications
+      - install -m644 res/com.gamestarmechanic.app.metainfo.xml -Dt ${FLATPAK_DEST}/share/metainfo
+      # Tweak desktop file values
+      - desktop-file-edit --set-key="Exec" --set-value="gamestarmechanic-wrapper %u" ${FLATPAK_DEST}/share/applications/com.gamestarmechanic.app.desktop
+  - name: libpepflashplayer
+    buildsystem: simple
+    sources:
+      - type: script
+        dest-filename: apply_extra
+        commands:
+          - mkdir plugins
+          - tar -zx -f flash_player_ppapi_linux.x86_64.tar.gz -C plugins libpepflashplayer.so
```

Comment:
so how is this gonna work on `aarch64`.

Reply:
Thank you for spotting that! The newest commit, alas, limits builds to x86_64.